code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -207,6 +207,16 @@
 	tastes = list("cobwebs" = 1, "sugar" = 2)
 	foodtype = JUNKFOOD | SUGAR
 
+/obj/item/reagent_containers/food/snacks/tobiko
+	name = "Tobiko"
+	desc = "Spider eggs wrapped in a thin salted Kudzu pod"
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_egg"
+	list_reagents = list("nutriment" = 6, "vitamin" = 2)
+	filling_color = "#FF3333" // R225 G051 B051
+	tastes = list("seaweed" = 1, "salty" = 2)
```

Comment:
but there's no seaweed in it?

Reply:
Kuz pods is the idea behind it, weed like taste + salty 1 + toxins from the eggs was the idea